themes/scalereal/static/css/style.css

```diff
@@ -3363,4 +3363,178 @@ lottie-player {
 
 .feedback-mobile  .owl-carousel .owl-dots {
   margin-top: -2px;
+}
+
+/* Client Success Stories Section */
+.client-success-stories-section .container .box {
+  padding-left: 98px;
+  padding-right: 98px;
+  width: 1190px;
+  max-width: 100%; 
+}
+.client-success-stories-section .box::before {
+  content: "";
+  position: absolute;
+  left: 48%;
+  top: 85px;
+  bottom: 85px;
+  right: 642;
+  width: 2px;
+  height: 70%;
+  background-color: #cccbcb; 
+}
+.client-success-stories-section .container .box .heading.box-content{
+  padding: 0px;  
+  margin-right: 25px;
+  margin-top: 100px;
+  padding-left: 0px;
+}
+.client-success-stories-section .container .box .heading.box-content h2{
+  padding-right: 86px;
+   
+}
+.client-success-stories-section .container .box .box-content{
+  margin: 0px;
+  padding-left: 3px;
+  width: auto;
+}
+.client-success-stories-section .container .box .box-content .owl-one .owl-dots {
+  margin-top: 24px;
+  align-items: center; 
+  margin-left: 33%;
+}
+.client-success-stories-section .container .box .box-content p{
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+  max-width: 423px;
+}
+.client-success-stories-section .container .box .box-content h3{
+  padding-top: 0px;

```

Comment:
whats the reason of mentioning padding-top and margin-top 0? can we remove this?